src/components/Card/index.js

```diff
@@ -103,7 +105,10 @@ const CardBody = ({title, instructor, type}) => {
   return (
     <div className={`${type === 'playlist' ? 'ph4' : ''}`}>
       <h3 className={titleHeadingClasses}>{title}</h3>
-      <div className={instructorNameClasses}>{instructor}</div>
+      <div className='mb4 flex justify-center items-center'>
+        <img src={instructorPortrait} alt='' className='w2 h2 br-pill mr3 pointer'/>
```

Comment:
Should I put the `instructorPortrait` classes to something like `instructorPortraitClasses`?